gr-blocks/lib/message_debug_impl.cc

```diff
@@ -20,16 +20,36 @@
 namespace gr {
 namespace blocks {
 
-message_debug::sptr message_debug::make()
+message_debug::sptr message_debug::make(bool en_uvec)
 {
-    return gnuradio::get_initial_sptr(new message_debug_impl());
+    return gnuradio::get_initial_sptr(new message_debug_impl(en_uvec));
 }
 
+message_debug_impl::message_debug_impl(bool en_uvec)
+    : block("message_debug", io_signature::make(0, 0, 0), io_signature::make(0, 0, 0)),
+      d_en_uvec(en_uvec)
+{
+    message_port_register_in(pmt::mp("print"));
+    set_msg_handler(pmt::mp("print"), boost::bind(&message_debug_impl::print, this, _1));
```

Comment:
Can these `boost::bind` be lambdas instead?

See https://github.com/gnuradio/gnuradio/pull/3441

Reply:
That's pretty neat; i hadn't seen that. I have updated this block to use lambdas.